charts/rancher-vsphere-cpi/rancher-vsphere-cpi/1.0.100/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 annotations:
-  catalog.cattle.io/rancher-version: <2.5.99-0
+  catalog.cattle.io/rancher-version: '>= 2.5.0-0 <= 2.5.99-0'
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: vSphere CPI
   catalog.cattle.io/namespace: kube-system
```

Comment:
To be clear, we are introducing 1.0.1, but 1.0.100 already exists ? So we are introducing a version for older cpi line ?

Reply:
1.0.1 is a copy of 1.0.000 with a fix to support k8s 1.17 clusters and a kubeversion to limit it, meant to replace 1.0.000. 1.0.100 is an entirely new chart based on a newer upstream. 1.0.100 should've been 1.1.0, but since it's already been released, this was the only way to have both.